### ISSUE TITLE
[BUG] Fullnode synced but Wallet not synced 

### ISSUE THREAD

**Issue by ziomik** [Chia-Network/chia-blockchain#5573]
**Describe the bug**
I have a fullnode on Windows and it just does plotting while a local drive under linux does ahrvering. The full node is synchronized but the wallet says Not synced (both from the GUI and from the terminal).
Restarting the fullnode does not solve.
Everything works fine (this is confirmed by a block taken last night from the logs and from another spot outside the fullsync network)

**To Reproduce**
I do not know

**Expected behavior**
I do not know

**Screenshots**
https://prnt.sc/135ipvl
https://prnt.sc/135ir6y

- OS: [Windows and Linux]

**Comment by skxpink** [comment]
same here, 8444 open, v1.1.5, win10

**Comment by MustDhiMust** [comment]
try add introducer node

introducer-or.chia.net

**Comment by wu-dennis** [comment]
add introducer node
2.238.73.140
51.154.10.127

**Comment by ziomik** [comment]
Done but not needed.
Also because the fullnode is perfectly synchronized

**Comment by rvp8762446** [comment]
Same here:

- Everything worked fine while plotting on windows 10 with version 1.14
- After plotting was finished I moved plots to a PI4(8GB) with Ubuntu Desktop 21.04 to farm. When 1.15 came out, I upgraded. Farming worked fine, node synced fine, wallet synced fine. I have plenty of bandwidth (1000Mbit Down/50Mbit Up and not being used much for other purposes)
- Somehow I was getting issue with Ubuntu Desktop 21.04 on the Pi (desktop was not loading properly after login, CLI was still working though). I think this happened after I did some updates en autoremove. Reinstalled a few times, same issue reoccured. So I decided to install Raspbian 64 instead.
- By now network grew a lot and syncing was really hard. I did backup ' blockchain_v1_mainnet.sqlite' which won some time, but I did not have the wallet blockchain_wallet_v1_mainnet_*.sqlite backed up, so this needed to sync from scratch. Which it did for a couple of days, but then it stopped syncing somehow.
- Tried a lot of things and some worked for a small time (deleting .chia/mainnet/db and or wallet etc and starting over etc.). Sometimes Node was synced again so I could farm a bit, but eventually it would go out of sync again and wallet stopped syncing altogether, never managed to sync the wallet again...
- I have port 8444 open. Tried opening/closing 8444 - disabling/enabling UPnP etc etc. same issue occured everytime: node syncing but eventually having hard time to keep up, wallet stops syncing early on. Also tried reducing amounts set in config.yaml regarding outbound_peer and peer count, buit did not notice any clear difference, even with low numbers. target_outbound_peer_count: 8  target_peer_count: 80
- Decided to reinstall again, now using Ubuntu Server 20.04 LTS. Seemed to work again, but next morning node did still not catch up and wallet stopped syncing at Wallet height 4735.
- I do see errors reoccuring several times in debug.log I can not explain myself, most recent one concerning wallet: 2021-05-20T08:52:34.859 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 775-807. Traceback (most recent call last):

- I have not been able to farm for almost a week now because of the syncing issues, meanwhile the network grew much faster and larger, so probably not going to be able to find a block anyway with just around 100 plots with big players taking ownership of large parts of the network.
- I don't have a large sum of Chia, I just bought a very small amount to test my wallet, but because it is not syncing, I can not even access that amount anymore, so the basic function of having Chia and being able to make crypto transferrals is not possible either.

- Probably paranoid but,  I am almost thinking some technically more advanced people are deliberately trying to brake this network for other/small time farmers so their own chances increase even further. If that is the case, this whole project would be flawed, so I am probably just being paranoid. I just hope there will be some new release soon which is more stable and fixes wallet and node syncing issues so many of us are having apparantly.

- In a last desperate attempt, I have now installed chia on my windows machine again as well and the wallet is syncing.... If it ever finishes I hope to copy blockchain_wallet_v1_mainnet_*.sqlite to my ubuntu in attempt to get it synced and over there as well. 

- Meanwhile I still hope the Node catches up on the  Ubuntu install by itself so it starts farming again. If this does not work, I am giving up, at least untill a new version is released which may provice better stability and hopefully fixes syncing issues. If these issues remain even after a new release, I think I will give up on Chia all together, but I hope it doesn't come to that.

**Comment by skxpink** [comment]
> 
> 
> try add introducer node
> 
> introducer-or.chia.net

done but nope

**Comment by skxpink** [comment]
> 
> 
> try add introducer node
> 
> introducer-or.chia.net

done but nope

**Comment by skxpink** [comment]
> 
> 
> add introducer node
> 2.238.73.140
> 51.154.10.127

done but nope

**Comment by rvp8762446** [comment]
> 
> 
> Same here:
> 
>     * Everything worked fine while plotting on windows 10 with version 1.14
> 
>     * After plotting was finished I moved plots to a PI4(8GB) with Ubuntu Desktop 21.04 to farm. When 1.15 came out, I upgraded. Farming worked fine, node synced fine, wallet synced fine. I have plenty of bandwidth (1000Mbit Down/50Mbit Up and not being used much for other purposes)
> 
>     * Somehow I was getting issue with Ubuntu Desktop 21.04 on the Pi (desktop was not loading properly after login, CLI was still working though). I think this happened after I did some updates and autoremoves. Reinstalled a few times, same issue reoccured. So I decided to install Raspbian 64 instead.
> 
>     * By now network grew a lot and syncing was really hard. I did backup ' blockchain_v1_mainnet.sqlite' which won some time, but I did not have the wallet blockchain_wallet_v1_mainnet_*.sqlite backed up, so this needed to sync from scratch. Which it did for a couple of days, but then it stopped syncing somehow.
> 
>     * Tried a lot of things and some worked for a small time (deleting .chia/mainnet/db and or wallet etc and starting over etc.). Sometimes Node was synced again so I could farm a bit, but eventually it would go out of sync again and wallet stopped syncing altogether, never managed to sync the wallet again...
> 
>     * I have port 8444 open. Tried opening/closing 8444 - disabling/enabling UPnP etc etc. same issue occured everytime: node syncing but eventually having hard time to keep up, wallet stops syncing early on. Also tried reducing amounts set in config.yaml regarding outbound_peer and peer count, but did not notice any clear difference, even with low numbers. So now using 1.15 defaults again (target_outbound_peer_count: 8  target_peer_count: 80)
> 
>     * Decided to reinstall again, now using Ubuntu Server 20.04 LTS. Seemed to work again, but next morning node did still not catch up and wallet stopped syncing at Wallet height 4735.
> 
>     * I do see errors reoccuring several times in debug.log I can not explain myself, most recent one concerning wallet: 2021-05-20T08:52:34.859 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 775-807. Traceback (most recent call last):
> 
>     * I have not been able to farm for almost a week now because of the syncing issues and the downtime trying to address these, meanwhile the network grew much faster and larger, so probably not going to be able to find a block anyway with just around 100 plots with big players taking ownership of large parts of the network.
> 
>     * I don't have a large sum of Chia, I just bought a very small amount to test my wallet, but because it is not syncing, I can not even access that amount anymore, so the basic function of having Chia and being able to make crypto transferrals is not possible either.
> 
>     * Probably paranoid but,  I am almost thinking some technically more advanced people are deliberately trying to brake this network for other/small time farmers so their own chances increase even further. If that is the case, this whole project would be flawed, so I am probably just being paranoid. I just hope there will be some new release soon which is more stable and fixes wallet and node syncing issues so many of us are having apparantly.
> 
>     * In a last desperate attempt, I have now installed chia on my windows machine again as well and the wallet is syncing.... If it ever finishes I hope to copy blockchain_wallet_v1_mainnet_*.sqlite to my ubuntu in attempt to get it synced and over there as well.
> 
>     * Meanwhile I still hope the Node catches up on the  Ubuntu install by itself so it starts farming again. If this does not work, I am giving up, at least untill a new version is released which may provice better stability and hopefully fixes syncing issues. If these issues remain even after a new release, I think I will give up on Chia all together, but I hope it doesn't come to that.

Small update: so far Chia on the windows machine, the full node synced in no-time and seems to stay synced so far as well... (UpnP true in config.yaml on windows machine, false on Pi). The wallet is also constantly syncing fine it seems and surpassed the height where it stopped on the Pi after one night, in just about 30 minutes or so. I should check if it continues to function well of course, but it looks much faster/better working than on Pi already....

Conclusion must be something is wrong on the Pi setup on my end I think than?
It is connected wireless (although speed/bandwidth/packet loss does not seem to be an issue), runs on a 32GB Transcend SSD. UFW is installed/enabled and only allows SSH from within a certain IP on local subnet and 8884. Could it be a hardware or config problem? Maybe the SSD is not fast enough or has writing issues which causes these issues? Should I connect wired only? Could it be that the Pi CPU is not fast enough to cope with all the syncing and causes issues? (plenty of mem. left, but CPU is at 100% all the time when syncing it seems). Could it be something with UFW config? Is farming on Pi just not advisable or should it work?

**Comment by jh442** [comment]
I have the same problem. I can get the full node to sync. but barely little to no progress on the wallet.  I've tried deleting the wallet db to start from scratch which did not appear to help

**Comment by rvp8762446** [comment]
I was happy to see a new release when I woke up. I just upgraded to 1.16 on both my windows machine and my Pi. Full node was syncing/stayed synced very fast and (after a few minutes!) the wallet started syncing on both machines as well. Wallet height on the Pi is growing much faster than before to it seems. 
So at first glance, this seems to fix the issue for me! Just have to wait and see how if the wallet can actually sync completely and both node and wallet can keep up with the network now, but I have a feeling it will, looking at how much faster syncing both node and wallet seems to go now.

**Comment by rvp8762446** [comment]
However... some time later: on my windows install I am having (only wallet, node is fine) I am running into syncing issues again: 
- Stopped syncing @202987
- Found following in log: 2021-05-21T08:38:59.304 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 203007-203039. Traceback (most recent call last):

-I stopped all Chia processes, reopened.  Now the GUI stated that is was stuck at 203007 suddenly (which makes more sense comparing to error I found in log..)
- waited a few minutes, and at some point it started syncing again fortunately.

So even release 1.1.6 seems to fix a lot of issues/improve things, it still seems to require constant monitoring.
